### PR TITLE
Initial role PR

### PR DESCRIPTION
**Pull request by haslersn**
* [x] Add `README.md`

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  company: stuvus (https://stuvus.uni-stuttgart.de)
+
+  license: AGPLv3
+
+  min_ansible_version: 2.4
+
+  platforms:
+    - name: Debian
+      versions:
+        - stretch
```
Comment (SlothOfAnarchy) [inline meta/main.yml:14]:
```suggestion
        - stretch
        - buster
```

File: tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      Package: *
+      Pin: release a=unstable
+      Pin-Priority: -1
+    dest: /etc/apt/preferences.d/limit-unstable
+
+- name: Install WireGuard
+  apt:
+    update_cache: True
+    name:
+      - wireguard
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]: missing newline

File: tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  copy:
+    content: |
+      Package: *
+      Pin: release a=unstable
+      Pin-Priority: -1
+    dest: /etc/apt/preferences.d/limit-unstable
+
+- name: Install WireGuard
+  apt:
+    update_cache: True
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]: I think the `apt_repository` module updates the apt cache on changes, so this may not be necessary.

File: tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+- name: Add unstable repo
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]:
```suggestion
- name: Add Debian unstable repo
```

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]
README and license file are missing

**Comment by haslersn** [comment]
> Why check this - shouldn't `interface` and `peer` be mandatory?

Now this role doesn't support configuring wireguard anymore. Configuring wireguard can be done with our [systemd-network role](https://github.com/stuvusIT/systemd-network).

File: tasks/main.yml (hunk 3 above)
Comment (haslersn) [reply tasks/main.yml]: Apparently [it does](https://docs.ansible.com/ansible/latest/modules/apt_repository_module.html). I will remove line 16.

**Comment by haslersn** [comment]
> README and license file are missing

`LICENSE` is already on master. `README.md` has now been added.

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**